tests/e2e/hci.go

```diff
@@ -210,87 +227,268 @@ var _ bool = ginkgo.Describe("hci", func() {
+	*/
+	ginkgo.It("A host down in one AZ", ginkgo.Label(p0, block, vanilla, hci), func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		var err error
+		hostIp := ""
+
+		scParameters = map[string]string{}
+		scParameters[scParamStoragePolicyName] = remoteStoragePolicyName
+		scSpec := getVSphereStorageClassSpec(defaultNginxStorageClassName, scParameters, nil, "", "", false)
+		sc, err := client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		defer func() {
+			err := client.StorageV1().StorageClasses().Delete(ctx, sc.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
+		ginkgo.By("Create 2 sts with 3 replica and 1 replica respectively")
+		sts1 := GetStatefulSetFromManifest(namespace)
+		sts2 := GetStatefulSetFromManifest(namespace)
+		var one int32 = 1
+		sts2.Spec.Replicas = &one
+		sts2.Name = "web-nginx"
+		sts2.Spec.Template.Labels["app"] = sts2.Name
+		sts2.Spec.Selector.MatchLabels["app"] = sts2.Name
+		framework.Logf("Creating statefulsets")
+		CreateStatefulSet(namespace, sts1, client)
+		CreateStatefulSet(namespace, sts2, client)
+		defer func() {
+			ginkgo.By(fmt.Sprintf("Deleting all statefulsets in namespace: %v", namespace))
+			fss.DeleteAllStatefulSets(ctx, client, namespace)
+		}()
+
+		replicas1 := *(sts1.Spec.Replicas)
+		replicas2 := *(sts2.Spec.Replicas)
+		// Waiting for pods status to be Ready
+
+		fss.WaitForStatusReadyReplicas(ctx, client, sts1, replicas1)
+		fss.WaitForStatusReadyReplicas(ctx, client, sts2, replicas2)
+		gomega.Expect(fss.CheckMount(ctx, client, sts1, mountPath)).NotTo(gomega.HaveOccurred())
+		gomega.Expect(fss.CheckMount(ctx, client, sts2, mountPath)).NotTo(gomega.HaveOccurred())
+
+		nodes, err := client.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		pods1 := fss.GetPodList(ctx, client, sts1)
+		pods2 := fss.GetPodList(ctx, client, sts2)
+		workersPodMap := make(map[string]int)
+		targetWorkerName := ""
+		max := 0
+		for _, pod := range append(pods1.Items, pods2.Items...) {
+			workersPodMap[pod.Spec.NodeName] += 1
+			if workersPodMap[pod.Spec.NodeName] > max {
+				targetWorkerName = pod.Spec.NodeName
+			}
+		}
+
+		ginkgo.By("Power off a host which has a k8s-worker with attached PVs in cluster1")
+		workerNode, err := client.CoreV1().Nodes().Get(ctx, targetWorkerName, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		targetHost := e2eVSphere.getHostFromVMReference(ctx, getHostMoref4K8sNode(
+			ctx, client, workerNode))
+		hostMoRef := vim25types.ManagedObjectReference{Type: "HostSystem", Value: targetHost.Value}
+		targetHostSystem = object.NewHostSystem(e2eVSphere.Client.Client, hostMoRef)
+		finder := find.NewFinder(e2eVSphere.Client.Client, false)
+		var datacenters []string
+		cfg, err := getConfig()
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
use this method to get datacenter list -- > GetDatacenters

Reply:
We are fetching datacenters from e2e conf generated in pipeline.